### PR TITLE
Move StructureDefinitions to fhir-api

### PR DESCRIPTION
**Pull request by simonvadee**

### CODE REVIEW

File: server/src/fhir/definitions.ts [hunk 1]
```diff
@@ -28,24 +27,38 @@ export const bootstrapDefinitions = async () => {
     }
 
     // Put obj in key value store
+    console.log('loaded', id)
```
Comment (Jasopaum) [inline server/src/fhir/definitions.ts]: You could remove this

File: server/src/resolvers/mapping/index.ts [hunk 2]
```diff
@@ -4,9 +4,11 @@ import {
   MAPPING_VERSION_1,
   MAPPING_VERSION_2,
   CURRENT_MAPPING_VERSION,
+  MAPPING_VERSION_3,
 } from '../../constants'
 import handleV1 from './v1'
 import handleV2 from './v2'
+import handleV3 from './v2'
```
Comment (Jasopaum) [inline server/src/resolvers/mapping/index.ts]: from './v2'

File: client/src/components/mapping/FhirMappingPanel/AddResource/index.tsx [hunk 3]
```diff
@@ -96,17 +90,30 @@ const AddResource = ({ callback }: Props) => {
     }
   );
 
+  React.useEffect(() => {
+    async function fetchDefinitions() {
```
Comment (Jasopaum) [inline client/src/components/mapping/FhirMappingPanel/AddResource/index.tsx]: T'aimes pas ce que t'as fait

File: server/src/fhir/definitions.ts [hunk 4]
```diff
@@ -28,24 +27,38 @@ export const bootstrapDefinitions = async () => {
...
+      params: {
+        _count: 1000, // TODO: use pagination
+      },
+    },
+  )
+  for (const def of standardDefinitions.items.map(
+    ({ _source }: any) => _source,
+  )) {
+    cacheDefinition(def)
   }
```
Comment (Jasopaum) [inline server/src/fhir/definitions.ts]: Would something like 
```
standardDefinitions.items
  .map(({ _source }: any) => _source)
  .map(d => cacheDefinition(d))
```
be clearer?

Comment (simonvadee) [reply server/src/fhir/definitions.ts]: yes!

File: server/src/resolvers/mapping/index.ts (hunk 2 above)
Comment (simonvadee) [reply server/src/resolvers/mapping/index.ts]: done

File: server/src/fhir/definitions.ts (hunk 1 above)
Comment (simonvadee) [reply server/src/fhir/definitions.ts]: i prefer to keep this for now

**Pull request merged**